openfl/component/director/director.py

```diff
@@ -60,17 +60,19 @@ def acknowledge_shard(self, shard_info: director_pb2.ShardInfo) -> bool:
 
     async def set_new_experiment(self, *, experiment_name: str, sender_name: str,
                                  tensor_dict: dict,
-                                 collaborator_names: typing.Iterable[str], data: bytes) -> bool:
+                                 collaborator_names: typing.Iterable[str],
+                                 data_file_path: str) -> bool:
```

Comment:
Will be better if all params use one style
```python
async def set_new_experiment(self, *, 
                             experiment_name: str, 
                             sender_name: str,
                             tensor_dict: dict,
                             collaborator_names: typing.Iterable[str],
                             data_file_path: str) -> bool:
```

Reply:
reformatted